pkg/command/storagepubliclink.go

```diff
@@ -82,10 +82,22 @@ func StoragePublicLink(cfg *config.Config) *cli.Command {
 						},
 						"services": map[string]interface{}{
 							"publicstorageprovider": map[string]interface{}{
-								"mount_path":   cfg.Reva.StoragePublicLink.MountPath,
-								"mount_id":     cfg.Reva.StoragePublicLink.MountID,
-								"driver_addr":  cfg.Reva.StoragePublicLink.PublicShareProviderAddr,
-								"gateway_addr": cfg.Reva.Gateway.URL,
+								"gateway_addr":       cfg.Reva.Gateway.URL,
+								"mount_path":         cfg.Reva.StoragePublicLink.MountPath,
+								"mount_id":           cfg.Reva.StoragePublicLink.MountID,
+								"driver_addr":        cfg.Reva.StoragePublicLink.PublicShareProviderAddr,
```

Comment:
rename setting to "public_share_provider_addr" and adjust also in reva ? "driver_addr" is too confusing